hell/adventcalendar/2025/17/index.md

```diff
@@ -1,35 +1,204 @@
 ---
-title: "Gently Apply Your Breaks"
-author: "Your Name"
-author_bio: "Your short bio"
+title: "Them’s the Breaks"
+author: "Tyler Sticka"
+author_bio: "Tyler Sticka is a creative director, designer, writer and artist from Portland, Oregon. He works with organizations to create and ship expressive, powerful and performant web applications and experiences. He co-owns Cloud Four, a tight-knit web design and development consultancy with an outsized impact. The browser is his favorite design tool, and he loves to draw."
 date: 2025-12-17
 author_links:
-  - label: "Site"
-    url: "https://linktoyourblog123.com"
-    link_label: "linktoyourblog123.com"
-intro: "<p>Short introductory text</p>"
+  - label: "Tyler’s homepage and blog"
+    url: "https://tylersticka.com"
+    link_label: "tylersticka.com"
+  - label: "Tyler’s company, Cloud Four"
+    url: "https://cloudfour.com"
+    link_label: "cloudfour.com"
+  - label: "Follow Tyler on Mastodon"
+    url: "https://social.lol/@tylersticka"
+    link_label: "@tylersticka@social.lol"
+intro: "<p>Tyler walks through several HTML options for managing mid-content line breaks: How they work, when they’re appropriate, and alternatives to consider.</p>"
 image: "advent25_17"
 ---
-Some text.
-Some text.
 
-Some text. Some text.
+On the web, it’s easy to take line breaks for granted.
 
-## Heading
+We get them for free between our headings, paragraphs, list items, `<div>` elements and more. We display them as-is in our code snippets thanks to `<pre>`. And most magically of all, our browsers insert breaks _automatically_ where lines of text (or other text-like “inline” elements) would otherwise outgrow their container.
 
-Some text.
+But sometimes, that isn’t enough.
 
-## Heading
+Some words are too long and continuous to break automatically. Some words can be “orphaned” onto their own, lonely line. Occasionally, our content demands an overt break; more often, our designs call for their addition or removal.
 
-Some text.
+So it makes sense that HTML provides a few options for managing mid-content breaks. Some famously overused, others less known or understood:
 
+- The “break” element, `<br>`
+- The “word break opportunity” element, `<wbr>`
+- The “soft hyphen” character, `&shy;`
+- The “non-breaking space” character, `&nbsp;`
 
-<p class="highlight"><strong>Note:</strong> Some text.</p>
+Let’s “break” down (nyuk, nyuk) those techniques: What they do, when they’re appropriate, and alternatives to consider.
```

Comment:
> nyuk, nyuk

Is this a Three Stooges reference? Maybe we can link to this or another appropriate resource. :) https://www.youtube.com/watch?v=AKtwlHV1-O8